src/components/date-range-picker/date-range-picker.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { PopoverModule } from '../../components/popover';
+import { IconModule } from '../icon';
+import { DateRangePickerModule } from './date-range-picker.module';
+
+
+
+@Component({
+    selector: 'ux-components-date-range-picker',
+    template: `
+        <ux-date-range-picker
+            [(start)]="start"
+            [(end)]="end"
+            [showTime]="showTime"
+            [showTimezone]="showTimezone"
+            [showSeconds]="showSeconds"
+            [showMeridian]="showMeridian"
+            [showSpinners]="showSpinners"
+            [showNowBtn]="showNowBtn"
+            [startTimezone]="startTimezone"
+            [endTimezone]="endTimezone"
+            (startChange)="onRangeChange()"
+            (endChange)="onRangeChange()"
+            (keydown.escape)="popover.hide(); input.focus()"
+            (startTimezoneChange)="onTimezoneChange(true, $event)"
+            (endTimezoneChange)="onTimezoneChange(false, $event)">
+        </ux-date-range-picker>
+    `
+})
+export class DateRangePickerComponent {
+
+    onRangeChange(): void { }
+    endChange(): void { }
+
+    start: Date;
+    end: Date;
+    date: string;
+    invalid: boolean = false;
+    showTime: boolean = false;
+    showTimezone: boolean = false;
+    showSeconds: boolean = false;
+    showMeridian: boolean = true;
+    showSpinners: boolean = true;
+    showNowBtn: boolean = false;
+    startTimezone: any = { name: 'GMT', offset: 0 };
+    endTimezone: any = { name: 'GMT', offset: 0 };
+}
+
+fdescribe('Date Range Picker', () => {
+    let component: DateRangePickerComponent;
```

Comment:
It might be good to add a rule to our linting to prevent this:
https://www.npmjs.com/package/tslint-defocus

Its easy to forget to remove the `f` while testing and its a common occurrence which could easily go unnoticed. At least with the linting rule in place the build will fail if we use `describe` or `fit` whereas we are currently only relying on the review to notice this.